tests/test_monkeypatch.py

```diff
@@ -1,17 +1,64 @@
+from django.contrib.sites.models import Site
+
+from cms import api
 from cms.cms_toolbars import LANGUAGE_MENU_IDENTIFIER
+from cms.extensions.extension_pool import ExtensionPool
 from cms.test_utils.testcases import CMSTestCase
 from cms.toolbar.toolbar import CMSToolbar
 from cms.toolbar.utils import get_object_edit_url
 from cms.utils.urlutils import admin_reverse
 
 from djangocms_versioning.plugin_rendering import VersionContentRenderer
+from djangocms_versioning.test_utils.extensions.models import (
+    TestPageExtension,
+    TestTitleExtension,
+)
 from djangocms_versioning.test_utils.factories import (
     PageContentFactory,
     PageVersionFactory,
     PollVersionFactory,
+    UserFactory,
 )
 
 
+class MonkeypatchExtensionTestCase(CMSTestCase):
+    def setUp(self):
+        user = UserFactory()
+        self.version = PageVersionFactory(content__language="en")
+        pagecontent = PageContentFactory(
+            page=self.version.content.page, language="de"
+        )
+        self.page = self.version.content.page
+        site = Site.objects.first()
+        self.new_page = self.page.copy(
+            site=site,
+            parent_node=self.page.node.parent,
+            translations=False,
+            permissions=False,
+            extensions=False,
+        )
+
+        new_page_content = api.create_title(
+            page=self.new_page,
+            language=pagecontent.language,
+            slug='slug',
+            path='/path/here',
+            title=pagecontent.title,
+            template=pagecontent.template,
+            created_by=user
+        )
+        self.new_page.title_cache[pagecontent.language] = new_page_content
+
+    def test_copy_extensions(self):
+        '''Try to copy the extension, without the monkeypatch this tests fails'''
```

Comment:
try not to add different styles of docstrings to keep the code consistent. It's an art to shape shift your own style to keep codebases consistent.

Reply:
The bane of my existence. ''' forever in my heart <3 (I will change this).